reports/historical/_gmmp_2015_importer.py

```diff
@@ -1648,6 +1648,398 @@ def import_79(self, sheet_data):
 
         return all_data
 
+    def _sheet_80_secondary_import(self, data, groups, medium, col_start, end_index, row_end=17):
```

Comment:
I take it that we'll be renaming this since it's being used in more sheets than just `80`, right?

Reply:
Yes @clemence... The clean up will tackle all of those.